src/ff7/camera.h

```diff
@@ -0,0 +1,42 @@
+/****************************************************************************/
+//    Copyright (C) 2021 Cosmos                                             //
```

Comment:
Its 2022 :d and you should use a real name anyone can claim to be "Cosmos"

Reply:
I updated it overall. Good catch, thanks!